doc/lib.rst

```diff
@@ -105,16 +105,16 @@ Collections
 * `heapqueue <heapqueue.html>`_
   Implementation of a heap data structure that can be used as a priority queue.
 
-* `intsets <intsets.html>`_
```

Comment:
maybe move under a section of deprecated modules?

Reply:
You mean create a new section? Are there any other modules that were deprecated (after 1.0), that should go there too then?

Reply:
on 2nd thought, maybe simply write:
```nim
* `intsets <intsets.html>`_
Deprecated: Efficient implementation of a set of ints as a sparse bit set.

* `packedsets <packedsets.html>`_
Efficient implementation of a set of ordinals as a sparse bit set.
```
(using your best judgement)